Makefile

```diff
@@ -1,48 +1,41 @@
 all: install
 
 ifeq ($(OS), Windows_NT)
-	Platform := "Windows"
+	Platform=Msys
+install:
+	mkdir $(USERPROFILE)\.cache 2> /dev/null
+	cp ani-cli $(WINDIR)/system32/ani-cli
+uninstall:
+	rm -rf $WINDIR/system32/ani-cli
 else
         Platform = $(shell uname -o)
 endif
 
-
 ifeq ($(Platform), GNU/Linux)
 	PREFIX := /usr/local
 install:
 	cp ani-cli $(DESTDIR)$(PREFIX)/bin/ani-cli
 	chmod 0755 $(DESTDIR)$(PREFIX)/bin/ani-cli
 uninstall:
 	rm -rf $(DESTDIR)$(PREFIX)/bin/ani-cli
-else ifeq ($(Platform), Darwin)
+else
+ifeq ($(Platform), Darwin)
 	PREFIX := /usr/local
 install:
 	cp ani-cli $(DESTDIR)$(PREFIX)/bin/ani-cli
 	chmod 0755 $(DESTDIR)$(PREFIX)/bin/ani-cli
 uninstall:
 	rm -rf $(DESTDIR)$(PREFIX)/bin/ani-cli
-else ifeq ($(Platform),Windows)
-install:
-	mkdir $USERPROFILE/.cache 2> /dev/null
-	cp ani-cli-win $WINDIR/system32/ani-cli
-uninstall:
-	rm -rf $WINDIR/system32/ani-cli
-else ifeq ($(Platform),Android)
-	TERMUX_BIN := /data/data/com.termux/files/usr/bin
-install:
-	cp ani-cli $(TERMUX_BIN)/ani-cli
-	chmod 0755 $(TERMUX_BIN)/ani-cli
-	@echo 'am start --user 0 -a android.intent.action.VIEW -d "$$2" -e "http-header-fields" "$$1" -n is.xyz.mpv/.MPVActivity' > $(TERMUX_BIN)/mpv
-	chmod +x $(TERMUX_BIN)/mpv
-	mkdir -p $(HOME)/.cache
-uninstall:
-	rm -rf $(TERMUX_BIN)/ani-cli
-	rm -rf $(TERMUX_BIN)/mpv
 else
+ifeq ($(Platform), Android)
+	TERMUX_BIN := /data/data/com.termux/files/usr/bin
 install:
-	@echo 'Failed to detect your operating system'
+	cp ani-cli $(DESTDIR)$(PREFIX)/bin/ani-cli
+	chmod 0755 $(DESTDIR)$(PREFIX)/bin/ani-cli
 uninstall:
-	@echo 'Failed to detect your operating system'
+	rm -rf $(DESTDIR)$(PREFIX)/bin/ani-cli
+endif
+endif
 endif
 
-.SILENT .PHONY: all install uninstall
```

Comment:
I can see how having no output is worse than too much output, but would say that especially for our users, a success or failure message would be easier to digest.
The one word `.SILENT` is easy enough to remove manually whenever we actually need to do debugging